objects/ship/avalishipdoor/avalishipdoor.object

```diff
@@ -9,7 +9,7 @@
   "shortdescription" : "Avali Ship Door",
   "race" : "avali",
 
-  "apexDescription" : "We use these doors on our ships... wait no we don't.",
+  "apexDescription" : "We use these doors on our ships... wait, no we don't.",
```

Comment:
Like, here, "Wait" needs capitalized because it's clearly part of another sentence